packages/react-components/src/atoms/Toggle.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import css from '@emotion/css';
+import { lineHeight, perRem } from '../pixels';
+import { steel, paper, tin, charcoal } from '../colors';
+
+const checkboxLabelStyles = css({
+  display: 'grid',
+  gridTemplateColumns: '1fr 40px 1fr',
+  height: `${54 / perRem}em`,
+  borderRadius: '27px',
+  border: `1px solid ${steel.rgb}`,
+  overflow: 'hidden',
+  backgroundColor: steel.rgb,
+  color: tin.rgb,
+  stroke: tin.rgb,
+});
+
+const buttonStyle = css({
+  borderRadius: '27px',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+
+const iconStyles = css({
+  display: 'inline-block',
+  width: `${lineHeight / perRem}em`,
+  height: `${lineHeight / perRem}em`,
+  paddingRight: `${6 / perRem}em`,
+});
+const activeStyles = css({
+  backgroundColor: paper.rgb,
+  color: charcoal.rgb,
+  stroke: charcoal.rgb,
+});
+const checkboxStyles = css({
+  display: 'none',
+  '&:not(:checked) ~ .left': activeStyles,
+  '&:checked ~ .right': activeStyles,
+});
+const buttonRight = css({
+  gridColumn: '2 / span 2',
+  gridRow: 1,
+});
+const buttonLeft = css({
+  gridColumn: '1 / span 2',
+  gridRow: 1,
+});
+
+interface ToggleProps {
+  defaultChecked?: 'left' | 'right';
+  leftButtonText: string;
+  leftButtonIcon: React.ReactElement<SVGElement>;
+  rightButtonText: string;
+  rightButtonIcon: React.ReactElement<SVGElement>;
+  onChange: () => void;
+}
+
+const Toggle: React.FC<ToggleProps> = ({
+  defaultChecked = 'left',
+  leftButtonIcon,
+  leftButtonText,
+  rightButtonIcon,
+  rightButtonText,
+  onChange,
+}) => (
+  <label css={checkboxLabelStyles}>
+    <input
+      type="checkbox"
+      defaultChecked={defaultChecked === 'right'}
```

Comment:
I still think this should be `checked`, not `defaultChecked`. Otherwise the component becomes uncontrolled, not (necessarily) in sync with the state of the app

Reply:
You can actually already see this happen in Storybook